iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/match/voter/AffOrgMatchVoterStrengthEstimator.java

```diff
@@ -405,9 +408,30 @@ private AffMatchingService createAffMatchingService() {
         affMatchingService.setAffMatchResultWriter(new SimpleAffMatchResultWriter());
         
         
+        AffMatchOrganizationAltNameFiller altNameFiller = createAffMatchOrganizationAltNameFiller();
+        affMatchingService.setAffMatchOrganizationAltNameFiller(altNameFiller);
+        
         return affMatchingService;
     }
     
+    private AffMatchOrganizationAltNameFiller createAffMatchOrganizationAltNameFiller() throws IOException {
+        
+        AffMatchOrganizationAltNameFiller altNameFiller = new AffMatchOrganizationAltNameFiller();
+        
+        List<String> altNamesCountryCodes = of(
```

Comment:
My other comments related to `altNamesCountryCodes` are applicable here as well.